### ISSUE TITLE
Significant differences between Emby and Kodi TV episode identification/presentation 

### ISSUE THREAD

**Issue by anoma** [MediaBrowser/Emby#3205]
I am in the slow process of porting from Kodi/mysql to Kodi/Emby and have noticed significant differences between how the two setups identify/present TV episodes.

I understand there will always be corner cases but I believe this may be a bug or at the very least an area worth discussing options to make the presentation more intuitive since the filesystem layout appears very "normal" to me.

I have attached a zip file ([MattHouston.zip](https://github.com/MediaBrowser/Emby/files/1822528/MattHouston.zip))  of a the fake set of episodes and folder structure along with the URL only tvshow.nfo to allow for easy replication of this setup. I have also prior to this had someone else replicate this issue using this zip file to confirm it is not a local issue.

The differences are striking and best shown with a couple of screen shots:

First Emby:

![Emby Season level](https://i.imgur.com/8Y4oMqD.jpg)

Kodi:

![Kodi Season level](https://i.imgur.com/uNJ2pQP.jpg)
![Kodi Episode level](https://i.imgur.com/b81KqOM.jpg)

The most obvious differences

- In Emby S01 has the content of filesystem S01 folder and also  the single ep listed in it.  However as well as this the single ep also listed as a season in its own right as well 
- In Emby the S02 single ep listed as a season  in its own right but there is no complete S02 folder

**Comment by anoma** [comment]
A follow up with an even simpler example having identical symptoms (with or without URL only nfo makes no difference):

![Simpler Example](https://i.imgur.com/s0g8VZx.png)

**Comment by cayars** [comment]
You're not following the naming conventions used for the paths so that is most likely why you are getting weird results.
You want to have:
/TV Shows/Matt Houston (1982)/
/TV Shows/Matt Houston (1982)/Season 0
   all season 0 files here
/TV Shows/Matt Houston (1982)/Season 1
   all season 1 files here
/TV Shows/Matt Houston (1982)/Season 2
   all season 2 files here
/TV Shows/Matt Houston (1982)/Season 3
   all season 3 files here

There is slight differences in how you can setup you libs but this has always worked perfectly for me on Emby, Plex and Kodi.

Carlo

**Comment by anoma** [comment]
You are right this is not the `naming conventions used for the paths` but at the same time it is a very very common naming convention (probably the most common if you think about it). I would suggest that 1:1 Kodi structure compatibility would be a sensible end goal for a product.

I am working around it here with manual effort now which is less than ideal but https://github.com/MediaBrowser/Emby/issues/3207 gives me hope this will be more powerful in the long term.
